src/components/button/Button.tsx

```diff
@@ -22,13 +24,20 @@ export interface ButtonIconType {
 
 const Button: React.FC<ButtonType> = (props) => {
 
-    const {children, variant = "primary", active = false, disabled = false, ...args} = props
+    const {children, variant = "normal", color = "primary", active = false, disabled = false, ...args} = props
     const icon = getChild(children, ButtonIcon)
     const content = getContent(children, ButtonIcon)
 
+    const getVariant = (): string => {
+        if (variant == "none") return "button__withoutAll";
+        if (variant == "outlined") return "button__withoutBorder";
+        if (variant == "filled") return "button__withoutBackground";
+        return "";
+    }
```

Comment:
This is static and does not need to be a function. What do you think about making this a mapping object?
```suggestion
    const variantMappings = {
      none: "button--without-all",
      outlined: "button--without-border",
      filled: "button--without-background"
    }
```

Then you can get the variant like this: `variantMappings[variant] ?? ""`